src/benchmarks/micro/corefx/System.Drawing/Perf_Graphics_DrawBeziers.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+
+namespace System.Drawing.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX)]
+    public class Perf_Graphics_DrawBeziers
+    {
+        private Bitmap _image;
+        private Pen _pen;
+        private Graphics _graphics;
+        private Point _point1, _point2, _point3, _point4;
+        private Point[] _points;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _image = new Bitmap(100, 100);
+            _pen = new Pen(Color.White);
+            _graphics = Graphics.FromImage(_image);
+
+            _point1 = new Point(10, 10);
+            _point2 = new Point(20, 1);
+            _point3 = new Point(35, 5);
+            _point4 = new Point(50, 10);
+
+            _points = new[] {_point1, _point2, _point3, _point4};
```

Comment:
the old benchmarks were using 4 points for `DrawBezier_Point` and array of **7 different** points for `DrawBezier_Points`

https://github.com/dotnet/corefx/blob/9940166bff52155600f25f15da13a83c982526d2/src/System.Drawing.Common/tests/Performance/Perf_Graphics_DrawBeziers.cs#L42

I unified that to make sure that we compare apples to apples (same points)